src/Illuminate/Database/Console/ShowCommand.php

```diff
@@ -45,7 +45,8 @@ public function handle(ConnectionResolverInterface $connections)
         $data = [
             'platform' => [
                 'config' => $this->getConfigFromDatabase($database),
-                'name' => $this->getConnectionName($connection, $database),
+                'name' => $connection->getName(),
+                'driver' => $connection->getDriverName(),
```

Comment:
New feature: Show connection name and the driver name on distinct rows. Removing the need for pretty names.

Reply:
adding new driver name is OK, but we still need pretty connection names.

Reply:
What was displayed is a pretty version of the database type, with the detection of MariaDB with MySQL driver. To me, the connection name is the name that is in the configuration. This is what I have changed.

I can a method `Grammar::getServerType`

Reply:
I think this method should be on the `Connection` class like your first commit, but only returns the connection pretty name, I suggest `Connection::getConnectionTitle()`:

```php
public function getConnectionTitle(): string
{
    return 'MySQL';
}
```

Reply:
This name is misleading. If there are 2 MySQL connections, they would get the same title.

Reply:
Just like before? we just want the connection name here right? just like connection version that would be the same for any mysql connection. the `db:show` already has `database` row to display the database name.

Reply:
What about `getDriverTitle()`?

Reply:
Ok for `getDriverTitle`, in that case, we don't need the new "Driver" key. Instead, name of the connection in the configuration would be useful.